### PR TITLE
Issue6198

### LINKED ISSUE #6198: MacOs and Linux are supported
Mac and Linux are supported, it still shows planning in this page.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 034ffda3-6ef7-13f4-6341-82ac3c11f455
* Version Independent ID: 5527ef3f-6361-b037-d9c3-3d520cf2ef2c
* Content: [Threat & Vulnerability Management supported operating systems - Windows security](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/tvm-supported-os)
* Content Source: [windows/security/threat-protection/microsoft-defender-atp/tvm-supported-os.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/microsoft-defender-atp/tvm-supported-os.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @DulceMontemayor
* Microsoft Alias: **dolmont**

### PR DESCRIPTION
**Pull request by konstruktoid**
- Fix `^M` line endings, https://help.github.com/en/github/using-git/configuring-git-to-handle-line-endings
- Update MacOS and Linux support, since it's preview products and subject to change, link to the Mac and Linux documentation. Closes #6198.

### CODE REVIEW

**Comment by illfated** [comment]
@konstruktoid : I trust you will take care of the file merge conflict. Looks very good otherwise.

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/microsoft-defender-atp/tvm-supported-os.md [hunk 1]
```diff
@@ -1,55 +1,57 @@
...
+ms.collection: M365-security-compliance
+ms.topic: article
+---
+# Threat & Vulnerability Management supported operating systems and platforms
+
+**Applies to:**
+
+- [Microsoft Defender Advanced Threat Protection (Microsoft Defender ATP)](https://go.microsoft.com/fwlink/p/?linkid=2069559)
+
+>Want to experience Microsoft Defender ATP? [Sign up for a free trial.](https://www.microsoft.com/microsoft-365/windows/microsoft-defender-atp?ocid=docs-wdatp-portaloverview-abovefoldlink)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/tvm-supported-os.md]:
```suggestion
>Want to experience Microsoft Defender ATP? [Sign up for a free trial](https://www.microsoft.com/microsoft-365/windows/microsoft-defender-atp?ocid=docs-wdatp-portaloverview-abovefoldlink).
```

**Comment by Dansimp** [comment]
Unfortunately, the TVM support for MacOS and Linux are not available at this point. For MacOS and Linux, we have Microsoft Defender ATP (which is the next gen protection, antivirus, and other capabilities). But at this point in time, Threat and Vulnerability Management for these platforms is not yet available.

**Comment by DulceMontemayor** [comment]
Hi konstruktoid, according to our TVM PM, there's no TVM for Mac/Linux workflows yet so we can't merge this.

Thanks!

**Comment by konstruktoid** [comment]
My bad, I understood it as `Threat and Vulnerability Management` was part of the `Microsoft Defender ATP`.

**Comment by illfated** [comment]
The document linting is nice, so if you would be so kind as to go ahead with a document linting PR, I think that will be good.

**Pull request closed without merging**